lib/batch/institutionalize_bids_jam.rb

```diff
@@ -0,0 +1,42 @@
+# Institutionalize "Feinberg Biomedical Data Science Day 2106" under 
+# "Biomedical Data Science Day"
+# Create new "Chicago Biomedical Informatic and Data Science Jam" collection, 
+# add member collection "Chicago Biomedical Informatics and Data Science Jam 2016"
+# and institutionalize under "Center for Data Science and Informatics"
+# https://github.com/galterlibrary/digital-repository/issues/706
+
+biomedical_data_science_day = Collection.find("ab570598-e497-4c81-9351-1aa316252682")
+data_science_day_2016 = Collection.find("fbbf335e-4c4a-4f84-8220-631a8014b3fd")
+
+biomedical_data_science_day.members << data_science_day_2016
+biomedical_data_science_day.save! 
+data_science_day_2016.convert_to_institutional(
+  "institutional-center-for-data-science-and-informatics-root",
+  biomedical_data_science_day.id
+)
+
+chicago_BIDS_Jam_2016 = Collection.find("a58d0b93-df89-4f10-9064-fa4e99b79bb1")
```

Comment:
This is minor, but it might make sense to assign this on line 35 since it's not used in the lines between here and there and it gets used immediately after. This is unlikely to need to change though and it's not hard to just check where the variable is used so it's really not a big deal.

Reply:
I like the suggestion. I know this is a one time deal, but I do refer to these a lot.